docs/source/Coding/Setting-up-PyCharm.md

```diff
@@ -1,57 +1,104 @@
 # Setting up PyCharm with Evennia
 
-[PyCharm](https://www.jetbrains.com/pycharm/) is a Python developer's IDE from Jetbrains available for Windows, Mac and Linux. It is a commercial product but offer free trials, a scaled-down community edition and also generous licenses for OSS projects like Evennia.
+[PyCharm](https://www.jetbrains.com/pycharm/) is a Python developer's IDE from Jetbrains available for Windows, Mac and Linux. 
+It is a commercial product but offer free trials, a scaled-down community edition and also generous licenses for OSS projects like Evennia.
 
-> This page was originally tested on Windows (so use Windows-style path examples), but should work the same for all platforms.
+First, download and install the IDE edition of your choosing.
+The community edition should have everything you need, 
+but the professional edition has integrated support for Django which can help.
 
-First, install Evennia on your local machine with [[Getting Started]]. If you're new to PyCharm, loading your project is as easy as selecting the `Open` option when PyCharm starts, and browsing to your game folder (the one created with `evennia --init`). We refer to it as `mygame` here. 
+## From an existing project
```

Comment:
I've found it helps reading with a short intro, such as 

    Use this if you want to use PyCharm with an already existing Evennia game.

Reply:
You mean to replace the header or to add below it?

Reply:
Add below it. An intro to the section.